balancer-js/examples/data/pool-gauges.ts

```diff
@@ -0,0 +1,26 @@
+import { Network } from '../../src/index';
+import { BalancerSDK } from '../../src/modules/sdk.module';
+
+const sdk = new BalancerSDK(
+  { 
```

Comment:
This line makes me think `.prettierrc` is not being followed. It would be nice to have that properly set, so we avoid unnecessary formatting changes on upcoming commits.

Reply:
That's ok, I have properly set prettier plugin.